AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -564,21 +564,21 @@
  * Text File View Controller
  */
 "createtextfile.title" = "Create Text File";
-"createtextfile.dismiss.confirmation.title" = "Dismiss?";
-"createtextfile.dismiss.confirmation.message" = "Are you sure you want to cancel the creation of this text file?\n\nAll work will be discarded.";
-"document.edit.button.discard" = "Discard Changes";
+"createtextfile.dismiss.confirmation.title" = "Cancel";
+"createtextfile.dismiss.confirmation.message" = "If you cancel the creation of this text file the work won't be saved.";
+"document.edit.button.discard" = "Cancel";
```

Comment:
@andyhealey26 Apple guidelines suggest not using the word "Cancel" in cases such as this, as it's a _destructive_ operation. "Cancel" usually implies "cancel the action I just initiated". Hence the specific wording "Discard Changes" that we had before.